sql/sqlserver/server.go

```diff
@@ -674,22 +676,25 @@ func (s *Server) processColumns(desc *structured.TableDescriptor,
 
 func (s *Server) processInsertRows(node parser.SelectStatement) (rows []sqlwire.Result_Row, err error) {
```

Comment:
FYI, this method is misnamed now. It should really be something like `processSelect`.